lib/chai/utils/overwriteMethod.js

```diff
@@ -46,7 +48,15 @@ module.exports = function (ctx, name, method) {
     _super = _method;
 
   ctx[name] = function () {
+    var keep_ssfi = flag(this, 'keep_ssfi');
+    var old_ssfi = flag(this, 'ssfi');
+    if (!keep_ssfi && old_ssfi)
+      flag(this, 'ssfi', ctx[name]);
+
+    flag(this, 'keep_ssfi', true);
```

Comment:
Hi @Turbo87, I just want to confirm I understood the whole thing here:

Whenever you overwrite a method you set `keep_ssfi` to true before calling `_super` so you won't get that call on the stack due to those checks you added to `addMethod`, `addChainableMethod` and `addProperty`, am I right?